app/routers/google_connect.py

```diff
@@ -0,0 +1,38 @@
+from fastapi import Depends, APIRouter, Request
+from starlette.responses import RedirectResponse
+from loguru import logger
+
+from app.dependencies import get_db
+from app.internal.google_connect import (
+    get_active_user,
+    get_credentials,
+    fetch_save_events
+)
+
+router = APIRouter(
+    prefix="/google",
+    tags=["sync"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get("/sync")
+async def google_sync(request: Request,
+                      session=Depends(get_db)) -> RedirectResponse:
+    '''Sync with Google - if user never synced with google this funcion will take
+    the user to a consent screen to use his google calendar data with the app.
+    '''
+
+    user = get_active_user(session)  # getting active user
+
+    # getting valid credentials
+    credentials = get_credentials(user=user, session=session)
+
+    if credentials is None:
+        # in case credentials is none, this is mean there isn't a client_secret
+        logger.error("GoogleSync isn't available - missing client_secret.json")
```

Comment:
Maybe you should also return something to the user?

Reply:
Done.